server/src/main/java/io/crate/replication/logical/LogicalReplicationService.java

```diff
@@ -151,6 +157,9 @@ private void onChangedSubscriptions(@Nullable SubscriptionsMetadata prevSubscrip
                     @Override
                     public void onResponse(AcknowledgedResponse acknowledgedResponse) {
                         LOGGER.debug("Acknowledged logical replication for subscription '{}'", subscriptionName);
+                        LOGGER.debug("Start tracking metadata on remote cluster '{}'", subscriptionName);
+                        indexMappingChangesTracker.addRemoteClusterToTrack(subscriptionName);
```

Comment:
This listener is actually passed down to LogicalReplicationService.afterReplicationStarted(...).